controllers/comments.controllers.js

```diff
@@ -0,0 +1,12 @@
+const { removeComment } = require("../models/comments.models")
+
+exports.deleteComment = (req,res,next) => {
+    const {comment_id} = req.params
+    removeComment(comment_id)
+    .then(()=>{
+        res.status(204).send()
+    })
+    .catch((err)=>{
+        next(err)
+    })
```

Comment:
There are two ways you could slightly shorten this down:
- `res.sendStatus(204);` on line 7
- `.catch(next);` on line 9